ashutosh7i.md

```diff
@@ -0,0 +1,51 @@
+---
+name: "@ashutosh7i"
+project: "Bagpack Security Tracker"
+---
+
+# Bagpack Security Tracker
+
+## Summary
+
+What are you going to build?
+i am calling it Bagpack Security Tracker, it is a Security system which will be used to protect bags & the content of bag.
+As a college student we carry valueable things like laptop & cash in our bag, Sometimes it gets stolen or lost😢.
```

Comment:
hey @ashutosh7i :) this grant/event is open only for high school students or younger so i'm sorry we won't be able to move forward with your project. if this was an error, please let me know! 

please note that if you are a high school student, we will require verification later on before sending the grant. thank you!

Reply:
i will be joining college or say university to be specific in February 2023 so i was writing in that sense, 
i am really excited about this event and don't want to Leave this opportunity, I request you to please reconsider my application.
Thank you
Regards
@ashutosh7i

Reply:
> i will be joining college or say university to be specific in February 2023 so i was writing in that sense, i am really excited about this event and don't want to Leave this opportunity, I request you to please reconsider my application. Thank you Regards @ashutosh7i

@ashutosh7i thank you so much for your interest but i'm sorry but this program is only for current high school students or younger. best of luck and hope you're able to find support and move forward with this project, it's super cool!